server/test/integration/post.spec.js

```diff
@@ -106,4 +106,80 @@ describe('Posts API', () => {
     expect(response2.status).toBe(404);
     done();
   });
+
+  it('tests the delete post method', async done => {
+    const postData = {
+      title: 'Test post',
+      body: 'This is the body for a test post',
+    };
+
+    const response = await supertest(app)
+      .post('/posts')
+      .send(postData);
+
+    const createdPost = response.body;
+    const url = '/posts/';
+
+    const response1 = await supertest(app).delete(url.concat(createdPost._id));
+
+    expect(response1.status).toBe(200);
+    done();
```

Comment:
Should this also test that the post can longer be retrieved after it is deleted?